core/src/rpc.rs

```diff
@@ -202,6 +215,16 @@ pub trait RpcSol {
 
     #[rpc(meta, name = "fullnodeExit")]
     fn fullnode_exit(&self, _: Self::Metadata) -> Result<bool>;
+
+    #[rpc(meta, name = "getSignatureConfirmations")]
```

Comment:
@aeyakovenko if you are planning on this API changing in the future (seems like yes), then let's make it clear that this API is a work in progress by naming it something like "x-getSignatureConfirmations".   Alternatively a name such as "GetNumBlocksSinceSignatureConfirmation" seems closer to what the current API actually does.  wdyt?